### ISSUE TITLE
v3 - Issue if title of a justified nav item is to long

### ISSUE THREAD

**Issue by TheZoker** [twbs/bootstrap#17380]
If you justify a nav-tab and the content of a tab is to long, there is a style issue
![img_2015 08 28_11h26m52s_001_](https://cloud.githubusercontent.com/assets/1368405/9543253/ba9d48a4-4d77-11e5-84f3-e0c645791a10.png)

https://jsbin.com/tejaza/edit?html,output

BTW: Are justified navs removed in v4?

**Comment by cvrebert** [comment]
> BTW: Are justified navs removed in v4?

Yes: http://v4-alpha.getbootstrap.com/migration/#whats-removed

**Comment by TheZoker** [comment]
@cvrebert Any reason why?

**Comment by cvrebert** [comment]
Moving the `border-bottom` from `.nav-tabs.nav-justified>li>a` to `.nav-tabs.nav-justified>li:not(.active)` basically fixes this, at the cost of not working in IE8.

**Comment by cvrebert** [comment]
I presume @mdo was the one who removed it and would therefore know why.

**Comment by mdo** [comment]
There's no way around this kind of stuff with a justified nav. It's an equal-width set of nav links—if one of the nav links has far too much content, it simply doesn't work. There's no adjustments we can or should make as far as I'm concerned. Your only options are overriding the CSS yourself, or shortening the label.

And justified navs were removed, but might come back before launch. I rewrote all our nav stuff—and still have more work to do there—and haven't finished it all up yet.
